index_old.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
This file shouldn't be needed — that's a big part of the purpose of source control, after all!